sources/logs.datatables.php

```diff
@@ -208,53 +207,48 @@
 } elseif (isset($params['action']) && $params['action'] === 'access') {
     //Columns name
     $aColumns = ['l.date', 'i.label', 'u.login'];
-    //Ordering
-    $order = $params['order'][0] ?? [];
-    if (isset($order['dir'], $order['column'])
-        && in_array($order['dir'], $aSortTypes, true)
-        && (int) $order['column'] !== 0
-        && $order['column'] !== 'asc') {
-
-        $columnIndex = filter_var($order['column'], FILTER_SANITIZE_NUMBER_INT);
-        $dir = filter_var($order['dir'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
-        $sOrder = 'ORDER BY ' . $aColumns[$columnIndex] . ' ' . $dir . ' ';
+
+    // Ordering
+    if (isset($aColumns[$order['column']]) === true) {
+        $orderColumn = $aColumns[$order['column']];
     } else {
-        $sOrder = 'ORDER BY ' . $aColumns[0] . ' DESC';
+        $orderColumn = $aColumns[0];
     }
```

Comment:
Same here: We can take `$orderColumn = $aColumns[0];` by default and remove the else.